.github/workflows/sanitizers.yml

```diff
@@ -0,0 +1,81 @@
+name: Sanitizers tests
+
+on:
+  workflow_call:
+    inputs:
+      # For more information about the supported sanitizers in Rust, see:
+      # https://rustc-dev-guide.rust-lang.org/sanitizers.html
+      rust-sanitizer:
+        required: false
+        default: 'address'
+        type: string
+        description: 'A sanitizer to build Rust code with. Possible values are: address, cfi, hwaddress, kcfi, leak, memory or thread'
+      # For more information about the supported sanitizers in LLVM, see `LLVM_USE_SANITIZER` option in:
+      # https://www.llvm.org/docs/CMake.html
+      llvm-sanitizer:
+        required: false
+        default: 'Address'
+        type: string
+        description: 'A sanitizer to build LLVM with. Possible values are Address, Memory, MemoryWithOrigins, Undefined, Thread, DataFlow, and Address;Undefined'
+      path:
+        required: false
+        type: string
+        description: 'Path filter of the era-compiler-tester. For example: tests/solidity/simple'
+      mode:
+        required: false
+        type: string
+        description: 'Mode filter for the era-compiler-tester. For example: Y+M3B3 0.8.26'
+  pull_request:
+
+jobs:
+  run-with-sanitizers:
+    runs-on: ci-runner-compiler
+    timeout-minutes: 720
+    container:
+      image: matterlabs/llvm_runner:ubuntu22-llvm17-latest
+      options: -m 110g
+    env:
+      TARGET: x86_64-unknown-linux-gnu
+    steps:
+
+      - name: Checkout compiler-tester
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Build LLVM
+        uses: matter-labs/era-compiler-ci/.github/actions/build-llvm@abalias-cpr-1052-run-compiler-tester-with-address-sanitizer
```

Comment:
Should this be removed?

Reply:
Yes, as soon as I finalize testing and merge `abalias-cpr-1052-run-compiler-tester-with-address-sanitizer` to the `main` for the LLVM builder and CI repositories.